protobuf-format.md

```diff
@@ -0,0 +1,133 @@
+# Protocol Buffers Event Format for CloudEvents  - Version 0.1
+
+## Abstract
+
+The Protocol Buffers Format for CloudEvents (CE) defines the encoding
+of CloudEvents in the Protocol Buffers binary format.
+
+## Status of this document
+
+This document is a working draft.
+
+## 1. Introduction
+
+This specification defines how the [Context
+Attributes](spec.md#context-attributes) Attributes defined in the
```

Comment:
the 2nd "Attributes" (after the link) is a repeat.

Reply:
Done